docs/send-data/kubernetes/troubleshoot-collection.md

```diff
@@ -395,6 +490,268 @@ kube-prometheus-stack:
+2024-02-13T09:06:56.817Z DEBUG [sumologic_mock::router::otlp] metrics => Sample { metric: "otelcol_process_runtime_heap_alloc_bytes", value: 50173608.0, labels: {"service_version": "v0.92.0-sumo-0", "node": "sumologic-kubernetes-collection", "service": "collection-sumologic-metadata-logs_collection-sumologic-otelcol-logs-headless", "prometheus_service": "collection-sumologic-metadata-logs", "service_name": "otelcol-sumo", "pod_labels_app": "collection-sumologic-otelcol-logs", "pod_labels_statefulset.kubernetes.io/pod-name": "collection-sumologic-otelcol-logs-0", "pod_labels_release": "collection", "_collector": "kubernetes", "service_instance_id": "9efeb490-0d19-49b0-8128-01ac2aabe1df", "statefulset": "collection-sumologic-otelcol-logs", "_origin": "kubernetes", "namespace": "sumologic", "pod_labels_chart": "sumologic-4.4.0-23-g067275958d", "pod": "collection-sumologic-otelcol-logs-0", "pod_labels_heritage": "Helm", "container": "otelcol", "pod_labels_controller-revision-hash": "collection-sumologic-otelcol-logs-6ff4df45b", "job": "collection-sumologic-metadata-logs", "cluster": "kubernetes", "endpoint": "otelcol-metrics"}, timestamp: 1707815215023 }
+```
+
+## Collecting Spans and Traces
+
+### Check instrumentation content
+
+You can print spans on stdout of instrumentation related pods, and validate if they are correct. It may happen that spans are ingested, but with different metadata than you expect them.
+
+In order to print them on stdout, two steps are required:
+
+1. Disable ingesting logs from instrumentation related pods. This is required to prevent logs ingest spike. Add the following configuration to `user-values.yaml`:
+
+   ```yaml
+   debug:
+     instrumentation:
+       otelcolInstrumentation:
+         stopLogsIngestion: true
+       tracesGateway:
+         stopLogsIngestion: true
+       tracesSampler:
+         stopLogsIngestion: true
+   ```
+
+   Then update your collection and wait for all logs collector pods to be redeployed.
+
+2. Enable printing spans on stdout for instrumentation related pods, by adding the following to `user-values.yaml`:
+
+   ```yaml
+   debug:
+     instrumentation:
+       otelcolInstrumentation:
+         print: true
+         stopLogsIngestion: true
+       tracesGateway:
+         print: true
+         stopLogsIngestion: true
+       tracesSampler:
+         print: true
+         stopLogsIngestion: true
+   ```
+
+3. To revert changes, please perform first step as it is, and after configuration has been propagated to all pods, you can remove all configuration added in this section from the `user-values.yaml`.
+
+:::note
+It's important to perform first step exactly as it is, especially waiting for all logs collector pods to apply new configuration. We want to avoid situation in which logs collector pods are picking up debugging logs and sending them to Sumo Logic, as it may increase your costs.
```

Comment:
```suggestion
It's important to perform the first step exactly as it is, especially waiting for all logs collector pods to apply new configuration. We want to avoid the situation in which logs collector pods are picking up debugging logs and sending them to Sumo Logic, as it may increase your costs.
```